c/dynalloc.c

```diff
@@ -834,6 +854,65 @@ int dynallocDataset(DynallocInputParms *inputParms, int *reasonCode) {
 
 }
 
+int dynallocNewDataset(int *reasonCode, DynallocNewTextUnit *setTextUnits, int TextUnitsSize) {
```

Comment:
Arguments (in your case `TextUnitsSize`) should start with a lower case letter.

Reply:
Resolved